packages/cli/src/lib/utils/record.ts

```diff
@@ -0,0 +1,8 @@
+export function recordable<T, K extends keyof T>(obj: T[]): Pick<T, K>[] {
+  return obj.map((o) =>
+    Object.entries(o).reduce(
+      (o, key) => ({...o, [key[0]]: key[1]}),
+      {} as Pick<T, K>
+    )
+  );
+}
```

Comment:
It may look over complicated but that's the cleanest way I found to:
1. consume a non-"string index signature" type and convert it to a Record type (string index signature type) accepted by CliUx.
2. keep the typing between the input and output object.